agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4415,6 +4415,20 @@ bool slave_thread::autoconfig_wsc_parse_m2_encrypted_settings(
 
     // Swap back to host byte order to read and use config_data
     config_data.class_swap();
+
+    // TODO tlvf should check this
```

Comment:
Correct - we should add a `_length_max` meta variable to TLVF and add this support. Can you open a task for it and refer to it in the comment?

Reply:
NVM - I'm adding support for this as well in #265